starter-code/public/new.html

```diff
@@ -85,9 +85,10 @@
     <script src="vendor/scripts/highlight.pack.js"></script>
     <script src="scripts/article.js"></script>
     <script src="scripts/articleView.js"></script>
-    <!-- TODO: Due to changes in article.js and articleView.js to attach their interior objects to a global 'app' object, we need to make changes to the contents of the following <script> tag: -->
+    <!-- ????: Due to changes in article.js and articleView.js to attach their interior objects to a global 'app' object, we need to make changes to the contents of the following <script> tag: -->
     <script>
       articleView.initNewArticlePage();
```

Comment:
I am getting the following console error: articleView.js:96 Uncaught ReferenceError: Article is not defined
    at Object.articleView.create (articleView.js:96)
    at new.html:91